src/BCrypt.Tests/BCrypt.cs

```diff
@@ -285,14 +285,14 @@ public unsafe void EncryptDecrypt_AesCcm()
                 fixed (byte* pTagBuffer = &tagBuffer[0])
                 fixed (byte* pNonce = &nonceBuffer[0])
                 {
-                    authInfo.pbNonce = new IntPtr(pNonce);
+                    authInfo.pbNonce = pNonce;
                     authInfo.cbNonce = nonceBuffer.Length;
-                    authInfo.pbTag = new IntPtr(pTagBuffer);
+                    authInfo.pbTag = pTagBuffer;
                     authInfo.cbTag = tagBuffer.Length;
 
                     // Mix up calling the IntPtr and native pointer overloads so we test both.
```

Comment:
True. The IntPtr property now has the name `pbTag_IntPtr`. I have a couple tests sprinkled to test those. But I had to pick some change to this line (either switch the property I'm assigning to or drop the IntPtr use). In this case, I guess I picked this one. I could have gone either way.